website/docs/usage/projects.md

```diff
@@ -108,6 +107,12 @@ even cloud storage such as GCS and S3. You can also fetch assets using git, by
 replacing the `url` string with a `git` block. spaCy will use Git's "sparse
 checkout" feature to avoid downloading the whole repository.
 
+Sometimes your project configuration may include large assets that you don't
+necessarily want to download every time you run `spacy project assets`. That's
```

Comment:
```suggestion
Sometimes your project configuration may include large assets that you don't
necessarily want to download when you run `spacy project assets`. That's
```
I'd like to remove the "every time" because typically, `spacy project assets` shouldn't be redownloading assets that were already downloaded before, so that repetitive nature is already prevented in general.

Reply:
(if the suggestion is accepted I'm afraid you'll have to re-run `prettier` as well, sorry!)